### PR TITLE
i18n: add japanese translation

### PR DESCRIPTION
**Pull request by MaikoTan**

### CODE REVIEW

File: src/locales/ja.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        - 仕事してる…
+      pending: 仕事してるが、その前に {0} つの絵が完遂するべきだ。
+      invalid-size: 向上モードは生成された画像のみに用いられる。img2img を使いたければ、「--enhance」を追加せずにこのコマンドを実行してください。
+      invalid-resolution: 無効な解像度。幅と高さが64の倍数である必要があります。
+      file-too-large: ファイルのサイズが大きすぎる。
+      unsupported-file-type: ファイルのタイプがサポートされていません。
+      download-error: 画像のダウンロードが成功していません。
+      unknown-error: 不明なエラーが発生しました。
+      response-error: 不明なエラーが発生しました ({0})。
+      empty-response: サーバーが無効な画像を返されました、後で試しください。
```
Comment (MaikoTan) [inline src/locales/ja.yml:42]:
```suggestion
      empty-response: サーバーが無効な画像を返されました、後で試してください。

```

File: src/locales/ja.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+commands:
+  novelai:
+    description: AI で絵を描く
+    usage: |-
+      コンマで区切られた英語タグを入力してください。例：1girl, red eyes, black hair。
+      モデルに用いられるタグは Danbooru とほとんど同じです。
+      本プロジェクトのレポジトリに Star：https://github.com/koishijs/novelai-bot
```
Comment (MaikoTan) [inline src/locales/ja.yml:7]:
```suggestion
      本プロジェクトのレポジトリにスターを：https://github.com/koishijs/novelai-bot

```

File: src/locales/ja.yml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+    options:
+      enhance: 向上 (enhance) モードを有効
+      model: モデルを設定
+      resolution: 画像解像度を設定
+      sampler: サンプラーを設定
+      seed: シード値を設定
+      steps: ステップ数を設定
+      scale: CFG スケールを設定
```
Comment (MaikoTan) [inline src/locales/ja.yml:16]:
```suggestion
      scale: CFG スケール値を設定

```

File: src/locales/ja.yml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+      strength: ノイズ除去強度を設定
+      noise: ノイズ強度を設定
+      undesired: ネガティブ タグを設定
```
Comment (MaikoTan) [inline src/locales/ja.yml:19]:
```suggestion
      undesired: ネガティブ・タグを設定

```

File: src/locales/ja.yml [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+    messages:
+      expect-prompt: タグを入力してください。
```
Comment (MaikoTan) [inline src/locales/ja.yml:22]:
```suggestion
      expect-prompt: プロンプトを入力してください。

```

File: src/locales/ja.yml [hunk 6]
```diff
@@ -0,0 +1,47 @@
+commands:
+  novelai:
+    description: AI で絵を描く
+    usage: |-
+      コンマで区切られた英語タグを入力してください。例：1girl, red eyes, black hair。
```
Comment (MaikoTan) [inline src/locales/ja.yml:5]:
```suggestion
      コンマで区切られた英語のプロンプトやタグを入力してください。例：1girl, red eyes, black hair。

```

Comment (MaikoTan) [reply src/locales/ja.yml:5]: @ilharp 「タグ」と「プロンプト」、どっちがいい？

File: src/locales/ja.yml (hunk 5 above)
Comment (MaikoTan) [reply src/locales/ja.yml:25]: こちらも

File: src/locales/ja-JP.yml [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+
+      strength: ノイズ除去強度を設定
+      noise: ノイズ強度を設定
+      undesired: ネガティブ・タグを設定
+
+
+    messages:
+      expect-prompt: 生成呪文を入力してください。
+      expect-image: 画像を入力してください。
+      latin-only: 英数字だけが入力可能。
```
Comment (MaikoTan) [inline src/locales/ja-JP.yml:27]:
```suggestion
      latin-only: 英数字だけが入力可能です。
```

File: src/locales/ja-JP.yml [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+      too-many-words: 入力したタグが多すぎる。
+      forbidden-word: 一部の入力したタグが禁止されている。
+      concurrent-jobs@random:
+        - 後でね～今、猫の手も借りたいなの！
+        - 描けるの、た、タブレットが起動できませんだから。
+        - じゃ、まず絵を教えて。
+      waiting@random:
+        - 私はプロ絵師だから、どんな絵でも描けるの。
+        - 仕事してる…
```
Comment (MaikoTan) [inline src/locales/ja-JP.yml:36]:
```suggestion
        - 仕事している…
```

File: src/locales/ja-JP.yml [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+      pending: 仕事してるが、その前に {0} つの絵が完遂するべきだ。
+      invalid-size: 向上モードは生成された画像のみに用いられる。img2img を使いたければ、「--enhance」を追加せずにこのコマンドを実行してください。
```
Comment (MaikoTan) [inline src/locales/ja-JP.yml:38]:
```suggestion
      invalid-size: 向上モードは AI 生成画像のみに用いられる。img2img (指定画像から生成) を使いたければ、「--enhance」を追加せずにコマンドを再実行してください。
```

File: src/locales/ja-JP.yml [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+      invalid-resolution: 無効な解像度。幅と高さが64の倍数である必要があります。
+      file-too-large: ファイルのサイズが大きすぎる。
+      unsupported-file-type: ファイルのタイプがサポートされていません。
+      download-error: 画像のダウンロードが成功していません。
```
Comment (MaikoTan) [inline src/locales/ja-JP.yml:42]:
```suggestion
      download-error: 画像のダウンロードに失敗しました。
```

File: src/locales/ja-JP.yml [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+      unknown-error: 不明なエラーが発生しました。
+      response-error: 不明なエラーが発生しました ({0})。
+      empty-response: サーバーが無効な画像を返されました、後で試してください。
+
```
Comment (MaikoTan) [inline src/locales/ja-JP.yml:46]:
```suggestion
```

File: src/locales/ja-JP.yml [hunk 12]
```diff
@@ -0,0 +1,51 @@
...
+      request-failed: リクエストが失敗しました ({0})，後で試してください。
+      request-timeout: リクエストがタイムアウトしました。
+      invalid-password: メイルアドレスやパスワードが間違っています。
```
Comment (MaikoTan) [inline src/locales/ja-JP.yml:49]:
```suggestion
      invalid-password: メールアドレスやパスワードが間違っています。
```

**Comment by MaikoTan** [comment]
I think it is good, but I need @ilharp Il-Harper-san to double check it first.

File: src/locales/ja-JP.yml [hunk 13]
```diff
@@ -0,0 +1,50 @@
+commands:
+  novelai:
+    description: AI で絵を描く
+    usage: |-
+      コンマで区切られた英語の生成呪文 (プロンプト) を入力してください。例：1girl, red eyes, black hair。
+      モデルに用いられるタグは Danbooru とほとんど同じです。
+      興味があったら、レポジトリにスターを付けてください：https://github.com/koishijs/novelai-bot
+
```
Comment (shigma) [inline src/locales/ja-JP.yml:8]: Extra blank line?

Comment (MaikoTan) [inline src/locales/ja-JP.yml:8]:
```suggestion
```

Oops

File: src/locales/ja-JP.yml [hunk 14]
```diff
@@ -0,0 +1,49 @@
...
+      sampler: サンプラーを指定
+      seed: シード値を設定
+      steps: ステップ数を設定
+      scale: CFG スケール値を設定
+
+      strength: ノイズ除去強度を設定
+      noise: ノイズ強度を設定
+      undesired: 反対呪文 (ネガティブプロンプト) を設定
+
+
```
Comment (MaikoTan) [inline src/locales/ja-JP.yml:22]:
```suggestion
      scale: CFG スケール値を設定
      strength: ノイズ除去強度を設定
      noise: ノイズ強度を設定
      undesired: 反対呪文 (ネガティブプロンプト) を設定

```

File: src/locales/ja-JP.yml [hunk 15]
```diff
@@ -0,0 +1,47 @@
...
+      concurrent-jobs@random:
+        - 後でね～今、猫の手も借りたいなの！
+        - 描けるの、た、タブレットが起動できませんだから。
+        - じゃ、まず絵を教えて。
+      waiting@random:
+        - 私はプロ絵師だから、どんな絵でも描けるの。
+        - 仕事している…
+      pending: 仕事している前に {0} つの絵が完遂するべきです。
+      invalid-size: 向上モードは AI 生成画像のみに用いられる。img2img (指定画像から生成) を使いたければ、「--enhance」を追加せずにコマンドを再実行してください。
+      invalid-resolution: 無効な解像度。幅と高さが64の倍数である必要があります。
```
Comment (MaikoTan) [inline src/locales/ja-JP.yml:36]:
```suggestion
      invalid-resolution: 無効な解像度。幅と高さが 64 の倍数である必要があります。
```

Should add spaces between number and kana?

File: src/locales/ja-JP.yml [hunk 16]
```diff
@@ -0,0 +1,47 @@
...
+      unsupported-file-type: ファイルのタイプがサポートされていません。
+      download-error: 画像のダウンロードに失敗しました。
+      unknown-error: 不明なエラーが発生しました。
+      response-error: 不明なエラーが発生しました ({0})。
+      empty-response: サーバーが無効な画像を返されました、後で試してください。
+      request-failed: リクエストが失敗しました ({0})，後で試してください。
+      request-timeout: リクエストがタイムアウトしました。
+      invalid-password: メールアドレスやパスワードが間違っています。
+      invalid-token: トークンが無効または期間切れしました、管理者に連絡してください。
+      unauthorized: トークンが認可されていない、または契約が期間切れてる、管理者に連絡してください。
```
Comment (MaikoTan) [inline src/locales/ja-JP.yml:47]: And these two lines sounds like wrong?

File: src/locales/ja-JP.yml [hunk 17]
```diff
@@ -0,0 +1,47 @@
...
+      file-too-large: ファイルのサイズが大きすぎる。
+      unsupported-file-type: ファイルのタイプがサポートされていません。
+      download-error: 画像のダウンロードに失敗しました。
+      unknown-error: 不明なエラーが発生しました。
+      response-error: 不明なエラーが発生しました ({0})。
+      empty-response: サーバーが無効な画像を返されました、後で試してください。
+      request-failed: リクエストが失敗しました ({0})，後で試してください。
+      request-timeout: リクエストがタイムアウトしました。
+      invalid-password: メールアドレスやパスワードが間違っています。
+      invalid-token: トークンが無効または期間切れしました、管理者に連絡してください。
```
Comment (MaikoTan) [inline src/locales/ja-JP.yml:46]:
```suggestion
      invalid-token: 期間切れたまたは無効なトークンです。管理者に連絡してください。
```

File: src/locales/ja-JP.yml (hunk 16 above)
Comment (MaikoTan) [inline src/locales/ja-JP.yml:47]:
```suggestion
      unauthorized: アカウント契約が期間切れるか、トークンが認可されていません。管理者に連絡してください。
```

**Review by ilharp — APPROVED** [review]
LGTM

**Comment by MaikoTan** [comment]
> LGTM
> 



DANKE !

**Review by shigma — COMMENTED** [review]
Also need to update `ctx.i18n()` in `index.ts`.

**Review by shigma — APPROVED** [review]

**Pull request merged**